### ISSUE TITLE
Toggle button used in vcs dialog inaccessible

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5899]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.673
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- look at the version control dialog (git or svn)
- they both have a control that looks a lot like a 2-item tab control (`LeftRightToggleButton`)

### Describe the problem in detail
- this control cannot be navigated to by keyboard, and has no annotations to give screen readers a hint about what it does
- even though it seems to be a tab control at first glance, it is actually a single button that navigates to a different view, and indicates which view is visible by rendering of the button
- another way to look at it, a Tab control switches a view that the tab control itself is not in; this control switches a view that contains this control (thus switches to a different instances of itself, albeit in the same location on screen)
- this doesn't map easily to any usual UI roles

<img width="859" alt="2019-12-16_14-32-31" src="https://user-images.githubusercontent.com/10569626/70949064-c0d5ee80-2011-11ea-8321-112b9539e1c5.png">

### Describe the behavior you expected
- either make this a true tab control, which would require it to be moved outside the region it is controlling, and also would require user to click on the correct tab to switch instead of clicking anywhere on the button as it currently does
- or consider using standard controls to better reflect what it does; perhaps a label indicating what the current view is, and a button reading "Show foo" which would navigate to the other view?

Mock up of the latter:

<img width="895" alt="2019-12-16_14-41-32" src="https://user-images.githubusercontent.com/10569626/70950209-cb45b780-2014-11ea-9f16-98fb21a74719.png">

This control is also used in the build output panes such as Source Cpp; haven't looked to see exactly how it works there, and also being used in the upcoming Find and Replace work (in that case it is being more of a true Tab control).

**Comment by kevinushey** [comment]
We could also use this UI from the environment pane as inspiration:

<img width="399" alt="Screen Shot 2019-12-17 at 10 07 42 AM" src="https://user-images.githubusercontent.com/1976582/71022333-157e7580-20b5-11ea-9b0a-031380afd1f8.png">

My main complaint about the new treatment is that it's not entirely clear to me that Changes is a label, as opposed to a piece of clickable UI, while Show History still is.

**Comment by gtritchie** [comment]
> We could also use this UI from the environment pane as inspiration:

That might be better, yes. 

> My main concern about the new treatment is that it's not entirely clear to me that Changes is a label, as opposed to a piece of clickable UI, while Show History still is.

True. Could make the title label ("Changes" in the example) be visually hidden, and just have the button for switching to the other mode and rely on the radically different appearance as sufficient explanation of what is being shown for sighted users.

**Comment by gtritchie** [comment]
Also, plan to change the window title to match the mode (another accessibility requirement). Currently it always says "RStudio - Review Changes", that should change to "RStudio - Git History" (or something like that) when the history mode is selected.
